astropy/coordinates/tests/test_angles.py

```diff
@@ -458,48 +461,64 @@ def test_negative_zero_hm():
 
 def test_negative_sixty_hm():
     # Test for HM parser
-    a = Angle('-00:60', u.hour)
+    with warnings.catch_warnings():
+        warnings.simplefilter('ignore', IllegalMinuteWarning)
+        a = Angle('-00:60', u.hour)
```

Comment:
Sorry to be a pain, but I think a better syntax would be:

```
with pytest.warns(IllegalMinuteWarning):
    a = Angle('-00:60', u.hour)
```

This is more concise and actually explicitly checks that the warning is emitted.

Reply:
Note that if you want to also check a specific message you can do e.g. (example taken from another test)

```
        with pytest.warns(UserWarning, match='Spacing is too small'):
```